molecule/renew/converge.yml

```diff
@@ -7,14 +7,31 @@
   hosts: all
   vars:
     ca_ca_host: ca_default
+    ca_ca_password: ChangeMe
+    # for CA server separate CA and cert client directories allow to trigger notify action on client certificate change
+    ca_certs_dir: /opt/certs
+    ca_cert: true
+    ca_server_cert: true
     ca_logstash: true
     ca_etcd: true
     ca_etcd_group: molecule
-    ca_ca_days: 3650
     ca_valid_time: +365d
     ca_check_valid_time: +1w
     ca_renew: true
-  tasks:
-    - name: "Include CA role"
-      include_role:
-        name: "{{ lookup('env', 'MOLECULE_PROJECT_DIRECTORY') | basename }}"
+
+  roles:
+    - role: "{{ lookup('env', 'MOLECULE_PROJECT_DIRECTORY') | basename }}"
+    - role: "{{ lookup('env', 'MOLECULE_PROJECT_DIRECTORY') | basename }}"
```

Comment:
Role is called twice. Once should be enough.

Reply:
We run verification tasks for two key algorithms to test `genrsa` and `genpkey` openssl subcommands.